oracle/.env

```diff
@@ -4,8 +4,8 @@ CHAIN_ID="sei-chain"
 # Cron tab for how frequently prices will be posted (ex: 10 seconds)
 CRONTAB="*/10 * * * * *"
 
-# List of markets the oracle will post prices for, against ust 
-MARKET_IDS="btc,luna,btc,eth,sol,osmo"
```

Comment:
can probably leave here if we want to support luna?

Reply:
yea i think either way is fine since this is just a var